app/views/shared/_project_tabs.html.erb

```diff
@@ -29,5 +29,6 @@
   <li class="<%= 'active' if active == 'settings' %>">
     <%= link_to "Settings", edit_project_path(project) %>
   </li>
+  <%= Samson::Hooks.render_views(:project_tabs_view, self, { project: @project, active: active }) %>
```

Comment:
Including the new hook with the PR that uses it makes it easier to review / change ...